rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -524,6 +524,127 @@ function createRuleComplianceTable(gridId, data, contextPath, refresh) {
 
 }
 
+/**
+ *
+ * This is the expected report table that we display on node details, for
+ * reports, when we don't have relevant information for compliance
+ * (for example when we get reports for the wrong configuration id).
+ *
+ * @param gridId
+ * @param data
+ * @param contextPath
+ * @param refresh
```

Comment:
either fill the param with meaningful explaination, or remove them. Here they don't give any meaningful data